README.md

```diff
@@ -10,29 +10,8 @@ Add the gem to your Gemfile:
 
 Run `bundle install` to install it.
 
-To add or create a user with token authentication run:
-`rails generate token_authenticate_me:install <model>`
-
-Replace `<model>` with the class name used for users. This will create the necessary migration files, and optionally create the model file if it does not exist.
-
-**Right now this gem only supports creating the authentication model `User`, so it is recommended to call `rails generate token_authenticate_me:install user`**
-
-Include TokenAuthenticateMe::TokenAuthentication into the application controller or any controllers that require authorization:
-````rb
-require 'token_authenticate_me/controllers/token_authenticateable'
```

Comment:
Suggest keeping this section just fixing the invalid parts as it is still needed for controller authentication.